test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -361,11 +361,18 @@ def test_conv2d_unary(self):
         Test pattern of conv2d with unary post ops (such as relu, sigmoid) with X86InductorQuantizer.
         Currently, only relu as unary post op is supported.
```

Comment:
should this comment be updated as well to reflect that not only relu is supported?

Reply:
Thanks for the comment and changed accordingly.